packages/ng/src/style/_components.scss

```diff
@@ -1,3 +1,4 @@
 @import
 
-  'components/input';
+  'components/input',
+  '../plugins/ngMaterial/general';
```

Comment:
why do you import this here ?, in `_components` ? we may distribute components that dont need to override ng material

Reply:
I put it there by default and forgot to double check afterwards. I have to try if we can import it directly into each component scss when needed, but I'm afraid some overrides (overlay and anything that end up into the overlay zone) might need to be imported globally.